packages/service-utils/src/core/authorize/client.ts

```diff
@@ -57,7 +57,7 @@ export function authorizeClient(
 
     return {
       authorized: false,
-      errorMessage: "The origin is not authorized for this key.",
+      errorMessage: `The domain: ${origin}, is not authorized for this key. Please update your key permissions on the Thirdweb Dashboard`,
```

Comment:
lowercase `thirdweb dashboard` pls :)

in general we try to keep `thirdweb` lowercase everywhere

Reply:
cool. will remember now :)